manifests/m/Mozilla/Firefox/98.0/Mozilla.Firefox.locale.en-US.yaml

```diff
@@ -0,0 +1,38 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.1.0.schema.json
+PackageIdentifier: Mozilla.Firefox
+PackageVersion: "98.0"
+PackageLocale: en-US
+Publisher: Mozilla
+PublisherUrl: https://www.mozilla.org/en-US/
+PublisherSupportUrl: https://support.mozilla.org/en-US/
+PrivacyUrl: https://www.mozilla.org/en-US/privacy/websites
+Author: Mozilla Foundation
+PackageName: Mozilla Firefox
+PackageUrl: https://www.mozilla.org/en-US/firefox/
+License: Mozilla Public License Version 2.0
+LicenseUrl: https://www.mozilla.org/en-US/MPL/2.0
+CopyrightUrl: https://www.mozilla.org/en-US/foundation/trademarks/policy
+ShortDescription: Mozilla Firefox is free and open source software, built by a community of thousands from all over the world.
+Description: |-
+  Firefox Browser, also known as Mozilla Firefox or simply Firefox, is a free and open-source web browser developed by the Mozilla Foundation and its subsidiary, the Mozilla Corporation.
+  Firefox uses the Gecko layout engine to render web pages, which implements current and anticipated web standards.
+  In 2017, Firefox began incorporating new technology under the code name Quantum to promote parallelism and a more intuitive user interface.
+  Firefox is officially available for Windows 7 or newer, macOS, and Linux.
+  Its unofficial ports are available for various Unix and Unix-like operating systems including FreeBSD, OpenBSD, NetBSD, illumos, and Solaris Unix.
+Moniker: firefox
+Tags:
+- browser
+- cross-platform
+- foss
+- gecko
+- quantum
+- spidermonkey
+- web
+- web-browser
+ReleaseNotes: |-
+  Fixed
+  • Security fix
```

Comment:
```suggestion
  New
  • Firefox has a new optimized download flow. Instead of prompting every time, files will download automatically. However, they can still be opened from the downloads panel with just one click.
  • Firefox allows users to choose from a number of built-in search engines to set as their default. In this release, some users who had previously configured a default engine might notice their default search engine has changed since Mozilla was unable to secure formal permission to continue including certain search engines in Firefox.
  
  Fixed
  • Now, you can set a default app to open a file type. Choose the application you want to use to open files of a specific type in your Firefox settings.
  • After updating to Firefox version 98, "Always ask" download actions will now be reset.
  • Various security fixes.
```